contracts/RegisterDomains.sol

```diff
@@ -0,0 +1,60 @@
+//SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.0;
+
+struct DomainDetails {
+    address controller;
+    uint deposit;
```

Comment:
Any reason to spend gas saving the `deposit` it it is all time the same?
-1 point